raiden/network/transport/matrix/rtc/web_rtc.py

```diff
@@ -531,7 +532,14 @@ def _initialize_web_rtc(self, partner_address: Address) -> None:
                 partner_address=to_checksum_address(partner_address),
             )
             conn.send_hangup_message()
-            self.close_connection(partner_address)
+            task = self.close_connection(partner_address)
+            if task is not None:
+                yield_future(task)
+
+    def get_channel_init_timeout(self) -> float:
+        """Returns the number of seconds to wait for a channel to be established."""
+        # pylint: disable=no-self-use
+        return 30.0
```

Comment:
I'd prefer putting constant values to a central point in the code and not inlined

Reply:
We still have a constant, it's just moved to a method.
This is to make it easier to override in the test where we reduce the time period so as not to wait too long and therefore not prolong the test run needlessly.

Reply:
Wouldnt it be also ok to override the cosntant? 
Nonetheless, for me it's fine to leave it like that but I would still prefer to have the constant at some central point and not inline somewhere in the code. 
Things like that caused a lot of trouble in the past.

Reply:
Yeah, I understand, however if we had a constant (as we had before), we'd have something like
```python
_FOO = 30.0
```
on the module level and then in the test we'd need to modify the module-private constant, which leads to abstraction leaks and associated problems.

So in this case I figured, wrapping the constant inside a public method is an elegant way to both
- have it centrally defined
- accessible/overrideable from tests

Reply:
IIRC, the way we dealt with this in the past was
- have a constant in `constants.py`
- have a fixture yielding (or pytest monkeypatching) the value
- parametrize that fixture

Reply:
Yes, I am aware of that approach, however, it has several disadvantages:
- it collects all constants in a single module and makes them public, which breaks down encapsulation;
   for instance, we end up with things like `raiden.constants.SDP_MLINE_INDEX_DEFAULT`
- it requires writing a new fixture and then using it, which is more work than is really necessary;
   the same effect can be achieved by patching directly, similarly to what I've done in this PR
- having too many fixtures, especially nested ones, makes it harder to reason about our tests; I've felt this multiple times

Reply:
In my experience, it's better to define constants in environment variables. This way it's easier to redefine them in different environments (test, CI, mainnet, testnet...)

That being said, I'm under the impression that this details could be ignored for now, and we can think of refactoring after we got this next release out. Correct me if I'm wrong, but this is probably not going to be a point of huge impact in the code, is it?